flink-connector/src/main/java/com/datastrato/gravitino/flink/connector/catalog/BaseCatalog.java

```diff
@@ -280,37 +287,36 @@ public void alterTable(ObjectPath objectPath, CatalogBaseTable catalogBaseTable,
   }
 
   @Override
-  public List<CatalogPartitionSpec> listPartitions(ObjectPath objectPath)
+  public List<CatalogPartitionSpec> listPartitions(ObjectPath tablePath)
       throws TableNotExistException, TableNotPartitionedException, CatalogException {
-    throw new UnsupportedOperationException();
```

Comment:
Could you use the partition interface in Gravitino?

Reply:
@FANNG1 I'd like to create an issue to fix it and add more integration tests. What do you think?

Reply:
could you create an issue?

Reply:
https://github.com/apache/gravitino/issues/4216